code/mob/living/silicon/ai.dm

```diff
@@ -1414,8 +1414,10 @@ or don't if it uses a custom topopen overlay
 	if (src.sleeping) src.sleeping = 0
 	src.delStatus("knockdown")
 
+#define AI_CHARGE_RATE 10
 /mob/living/silicon/ai/use_power()
 	..()
+	var/mult = (max(tick_spacing, TIME - last_life_tick) / tick_spacing)
 	var/turf/T = get_turf(src)
```

Comment:
Any reason not to pass `mult` as an argument from silicon/Life instead (and also mult-adjust other silicons)?